packages/@adobe/spectrum-css-temp/components/textfield/index.css

```diff
@@ -179,6 +179,10 @@ governing permissions and limitations under the License.
     /* Treat all quiet inputs and textareas the same */
     resize: none;
     overflow-y: hidden;
+
+    padding-top: var(--spectrum-textfield-quiet-padding-top);
```

Comment:
This scares me. Why are we adding padding to textfield?

Reply:
This overrides the padding set here: https://github.com/adobe/react-spectrum/blob/main/packages/@adobe/spectrum-css-temp/components/textfield/index.css#L55.
In the XD file for 5.3 textfields, quiet textfields have a padding bottom of 8px (9px -1px from the border) and padding top of 6px